protocol/app/upgrades/v7.0.0/upgrade.go

```diff
@@ -71,6 +81,12 @@ func CreateUpgradeHandler(
 		// Migrate vault quoting params to vault params.
 		migrateVaultQuotingParamsToVaultParams(sdkCtx, vaultKeeper)
 
+		// Set default tiers for affiliates.
+		setDefaultTiersForAffiliates(sdkCtx, affiliatesKeeper)
+
+		// Set the module version to 1 to skip init genesis during the first upgrade.
+		vm[affiliatetypes.ModuleName] = 1
```

Comment:
Why is this needed for `x/affiliate` which is a new module that didn't exist in v6?

Reply:
its because RunMigrations calls InitGenesis [here](https://github.com/cosmos/cosmos-sdk/blob/3856e777b4feca32747d785387d6dd8e0cb805c2/types/module/module.go#L699)

Reply:
which overwrites setDefaultTiersForAffiliates